src/DataAPI.jl

```diff
@@ -331,39 +329,36 @@ metadatakeys(::Any) = ()
 
 Set metadata for object `x` for key `key` to have value `value`
 and style `style` and return `x`.
-If `x` does not support setting metadata throw `ArgumentError`.
+If `x` does not support setting metadata throw error.
```

Comment:
Can you update other places for consistency?
```suggestion
Throw an error if `x` does not support setting metadata.
```

Reply:
Note that there are more places to update.

Reply:
OK